packages/rpc/package.json

```diff
@@ -52,24 +53,26 @@
 	"homepage": "https://discord.js.org",
 	"dependencies": {
 		"@discordjs/collection": "workspace:^",
-		"@sapphire/async-queue": "^1.3.2",
+		"@sapphire/async-queue": "^1.5.0",
 		"@sapphire/snowflake": "^3.2.2",
-		"discord-api-types": "^0.36.1",
+		"discord-api-types": "^0.37.11",
 		"discord.js": "workspace:^",
 		"tslib": "^2.4.0",
-		"undici": "^5.7.0"
+		"undici": "^5.11.0"
 	},
 	"devDependencies": {
-		"@discordjs/docgen": "workspace:^",
-		"@discordjs/scripts": "workspace:^",
-		"@favware/cliff-jumper": "^1.8.5",
-		"@microsoft/api-extractor": "^7.28.3",
-		"c8": "^7.11.3",
-		"eslint": "^8.19.0",
+		"@favware/cliff-jumper": "^1.8.8",
+		"@microsoft/api-extractor": "^7.32.0",
+		"@types/node": "^16.11.64",
+		"@vitest/coverage-c8": "^0.23.4",
+		"cross-env": "^7.0.3",
+		"downlevel-dts": "^0.10.1",
+		"eslint": "^8.24.0",
+		"eslint-config-neon": "^0.1.33",
 		"prettier": "^2.7.1",
-		"tsup": "^6.1.3",
-		"typescript": "^4.7.4",
-		"vitest": "^0.17.0"
+		"tsup": "^6.2.3",
+		"typescript": "^4.8.4",
+		"vitest": "^0.23.4"
 	},
```

Comment:
are all these dependencies such as undici really needed?

Reply:
we are applying changes from upstream / other packages